content/blog/2016/2016-01-05-new-website.adoc

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "A new Jenkins website"
+tags:
+- www
+author: rtyler
+---
+
+When link:https://github.com/rtyler[I] first started working on the Jenkins
+website (then called by a different name) I selected
+link:https://drupal.org[Drupal], an extensible content management system, to
+get the job done. Like Jenkins itself, Drupal is rather easy to set up, install
+plugins and authoring content is easily accomplished in a nice web UI. After
+seven years, Drupal has served us well but it is time to move on to something
+better suited for our needs.
+
+The general requirements for something newer were:
+
+. Easy to edit and create content
+. Changes to content should be tracked and reviewable
+. Any Jenkins contributor should be able to participate
+. Support mixed content types
+
+
+The consensus was that a statically-generated site, with raw content hosted on
+GitHub, would meet the majority of the "ease-of-use" type requirements. The
+remainder could be addressed depending on the implementing. A couple of years
```

Comment:
"depending on the implementing"? Did you mean "depending on the implementation"?